src/testing_system/internal/controller/server.py

```
@@ -0,0 +1,74 @@
+            raise HTTPException(status_code=500, detail=str(e))
+
+    @app.post("/experiments/run", status_code=202)
+    async def run_experiments(request: RunExperimentsRequest = Body(...)):
+        """Запускает выполнение экспериментов асинхронно (202 Accepted)."""
+        if orchestrator is None:
+            raise HTTPException(status_code=503, detail="Service not initialized")
+        path = request.experiments_path or config.get("testing_system", {}).get("loaders", {}).get("path")
+        if not path:
+            raise HTTPException(status_code=400, detail="No experiments path provided")
+
+        asyncio.create_task(orchestrator.execute_experiments(path))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)